hw20/roles/hw201/tasks/hw20_user.yml

```diff
@@ -0,0 +1,20 @@
+---
+#Optional add sudo priv. to user
+- name: Add sudo to user
+  set_fact: sudo="{{ sudo_groups }}"
+  when: sudo_groups is defined
+
+#Add user on hosts
+- name: Add the user 'hw20' with a sh shell and '/opt/hw20' home directory
+  ansible.builtin.user:
+   name: "{{ item.name }}"
+   shell: "{{ item.shell }}"
+   home: "{{ item.home }}"
+   groups: "{{ item.groups }},{{ sudo }}"
```

Comment:
The only thing. Better to include `,` inside sudo var to have this `"{{ item.groups }}{{ sudo }}"` where `sudo =",{{ sudo_groups }}"`